src/Model/Resolver/ResetPassword.php

```diff
@@ -57,6 +70,9 @@ public function resolve(
         $resetPasswordToken = $args['token'];
         $password = $args['password'];
         $passwordConfirmation = $args['password_confirmation'];
+        $customerId = $args['customer_id'];
+
+        $customerEmail = $this->customerRepository->getById($customerId)->getEmail();
```

Comment:
Please test what will happen if you pass the wrong `customer_id` parameter.
I believe this will fail and we should respond with the error message to the frontend.